src/routes/load/Load.test.tsx

```diff
@@ -22,8 +23,10 @@ describe('<Load>', () => {
 
     render(<Load />, customState)
 
-    expect(screen.getByText('Add existing Safe', { selector: 'h2' })).toBeInTheDocument()
-    expect(screen.getByTestId('load-safe-form')).toBeInTheDocument()
+    await waitFor(() => {
+      expect(screen.getByText('Add existing Safe', { selector: 'h2' })).toBeInTheDocument()
```

Comment:
I don't know if we have a testing 'style guide', but `render` returns the equivalent of `screen`. Should we import it or use the render return?

Reply:
After talking about this in slack we agree in to use `screen` 😊

see this doc for more details: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen